packages/block-editor/src/components/media-replace-flow/style.scss

```diff
@@ -0,0 +1,63 @@
+.components-dropdown-menu__popover:not(.is-mobile) {
+	z-index: z-index(".components-dropdown-menu__popover");
+}
+
+.components-dropdown-menu__menu {
```

Comment:
We should avoid targeting component classes that are not coming from this component.

Reply:
it doesn't appear to me these are needed in any way right now.